src/Clock.tsx

```diff
@@ -0,0 +1,67 @@
+import { Component } from 'react';
+
+type State = {
+  currentTime: string;
+};
+type Props = {
+  name: string;
+};
+
+function toFormattedTime(date: Date): string {
+  return date.toUTCString().slice(-12, -4);
+}
+
+export class Clock extends Component<Props, State> {
+  state: Readonly<State> = {
+    currentTime: toFormattedTime(new Date()),
```

Comment:
I think will be more logical to store full date object here, but it's not critical